pmpro-set-expiration-dates.php

```diff
@@ -21,18 +29,16 @@ function pmprosed_pmpro_membership_level_after_other_settings()
     else
         $set_expiration_date = "";
     ?>
-    <h3 class="topborder">Set Expiration Date</h3>
-    <p>To have this level expire on a specific date, enter it below in YYYY-MM-DD format. <strong>Note:</strong> You
-        must also set an expiration date above (e.g. 1 Year) which will be overwritten by the value below.</p>
+    <h3 class="topborder"><?php _e( 'Set Expiration Date', 'pmpro-set-expiration-dates' ); ?></h3>
+    <p><?php _e( 'To have this level expire on a specific date, enter it below in YYYY-MM-DD format. <strong>Note:</strong> You must also set an expiration date above (e.g. 1 Year) which will be overwritten by the value below.', 'pmpro-set-expiration-dates' ); ?></p>

```

Comment:
Try to avoid wrapping HTML elements inside i18n functions.

Reply:
updated to printf with html elements as variables in string